src/main/java/uk/gov/hmcts/sptribs/services/ccd/CaseApiService.java

```diff
@@ -105,15 +105,27 @@ private CaseDataContent getCaseDataContent(String authorization, CaseData caseDa
 
     private CaseDataContent getCaseDataContent(String authorization, CaseData caseData, Event eventEnum,
                                                String userId, String caseId, AppsConfig.AppsDetails appsDetails) {
+
         final CaseDataContent.CaseDataContentBuilder builder = CaseDataContent.builder().data(caseData);
+
         if (eventEnum.getEventType().equalsIgnoreCase(Event.UPDATE.getEventType())) {
+
             builder.event(uk.gov.hmcts.reform.ccd.client.model.Event.builder().id(appsDetails.getEventIds().getUpdateEvent()).build())
                 .eventToken(getEventTokenForUpdate(authorization, userId, appsDetails.getEventIds().getUpdateEvent(),
                                                    caseId, appsDetails));
+
         } else if (eventEnum.getEventType().equalsIgnoreCase(Event.SUBMIT.getEventType())) {
+
             builder.event(uk.gov.hmcts.reform.ccd.client.model.Event.builder().id(appsDetails.getEventIds().getSubmitEvent()).build())
                 .eventToken(getEventTokenForUpdate(authorization, userId, appsDetails.getEventIds().getSubmitEvent(),
                                                    caseId, appsDetails));
+
+        } else if (eventEnum.getEventType().equalsIgnoreCase(Event.UPDATE_CASE.getEventType())) {
```

Comment:
Minor Comment : 
Sonarcloud is displaying a couple of code smells.
And a test can be added for the Event.UPDATE_CASE this small part is shown as uncovered. 
Other than that all great.